params/params_testnet.go

```diff
@@ -49,6 +49,8 @@ var TestNetParams = Params{
 				MainHeight:      0,
 			},
 		},
+		// after this height the big graph will be the main pow graph
+		BigGraphStartHeight:45 * 1440 * 4,
```

Comment:
`BigGraphStartHeight` should depend on `TargetTimePerBlock` , and changed according to `TargetTimePerBlock` 60/15 = 4 block/min, and 1 day is 1440 mins